### ISSUE TITLE
Internationalizing the manual?

### ISSUE THREAD

**Issue by iku000888** [clojure-emacs/cider#1752]
Dear maintainers and contributors,

First off, thank you for your life changing and amazing work!

I created this issue because I want to brainstorm on how to proceed with internationalizing the manual.
(for me, it would be in Japanese). Would it be possible to set up an official/approved platform that allows people to contribute translations in any language? (i.e. transifex?)

I would work on translating the manual anyways,
but I wanted to hear what the community has in mind before I take off.
Please let me hear what you think! Thanks!

**Comment by bbatsov** [comment]
As far as I know ReadTheDocs has native support about multi-language manuals. The only problem is that often the translated versions would diverge from the English version, as they would typically be maintained by just a single person who may or not be available to always keep the translations up-to-date. 

I'm not sure whether the multi-language support will work with mkdocs, though. Someone will have to investigate this.

**Comment by iku000888** [comment]
@bbatsov Thanks for giving attention to this issue!

Agree that serious hammock time is needed.

On the "who would maintain" part, a model project could be
the python Japanese documentation project.
Basically, anybody interested can contribute translations on transifex
and the combined work would be released by the core maintainer.
Keeping things up to date is really easy because the barrier to contribute is basically creating a transifex account. https://github.com/mkdocs/mkdocs/issues/617

On the mkdocs part, I found https://github.com/mkdocs/mkdocs/issues/617.

The primary thought of this is to avoid replicating the same mechanism for n different languages.
I will keep post some more ideas as they emerge.

**Comment by bbatsov** [comment]
I'm guessing sphinx has support for multiple languages, but someone will have to migrate the current manual to sphinx first. That's not something I'm eager to tackle myself.

**Comment by iku000888** [comment]
As long as migrating to sphinx is fine with you,
I certainly do not mind tackling the migration.

Just out of curiosity, were there any particular reason for going with ReadTheDocs?

**Comment by iku000888** [comment]
There was also this issue back then.
https://github.com/clojure-emacs/cider/issues/689

**Comment by bbatsov** [comment]
> Just out of curiosity, were there any particular reason for going with ReadTheDocs?

It was easy to setup, has search (which works only with sphinx), has a notion of stable and master documentation and can even export the docs to pdf and epub. It also auto-builds the documentation on every commit. I haven't done much of a research - I just saw a lot of project were using it and it looked good enough.

**Comment by iku000888** [comment]
Thanks for the background/context.

Apparently ReadTheDocs can host sphinx projects,
So I am 99% convinced that migrating the docs to sphinx is the way to go.
I will sleep on the idea for a bit just in case something else pops up.

**Comment by bbatsov** [comment]
Yeah, ReadTheDocs is basically sphinx-centric. Unfortunately I didn't have time to learn it, so I opted to use `mkdocs`, as I know markdown pretty well and I didn't know how much of the functionality was broken under mkdocs...

**Comment by iku000888** [comment]
@bbatsov ,

I am thinking of getting my hands dirty with migrating the docs to sphinx format, still hosted on ReadTheDocs, after sleeping over the idea for a night.

Could you summarize/point-to how the current documentation build pipeline works?
I want to experiment with ReadTheDocs using a temporary fork, but would appreciate any advice as it is the first time even touching ReadTheDocs for me. (Do have some knowledge of sphinx, though).
Once I get the gist of it I would probably open a WIP PR.

Thanks!

**Comment by bbatsov** [comment]
Basically the docs are rebuilt automatically when changes are pushed to github. One can run mkdocs locally to see the result of some changes without pushing. I guess to migrate things to sphinx you'll simply have to run it locally as well and when you're ready I'll change the rtd config so it would know to build the proper type of documentation.

**Comment by bbatsov** [comment]
I'll close this due to lack of activity.
